create_derivatives/helpers.py

```diff
@@ -1,7 +1,36 @@
+from os import listdir
+from os.path import isfile, join
```

Comment:
I started out using `pathlib` in this file (and in the app in general). I think it would be better to stick with one approach rather than mixing and matching with `os`. https://docs.python.org/3.6/library/pathlib.html